documentation/introduction/tutorial-get-started-first-part.asciidoc

```diff
@@ -2,87 +2,103 @@
 title: Part 1 - How to use Vaadin Flow
 order: 2
 layout: page
-subnav_auto_list_numbers: true
```

Comment:
@ruthstocks I guess we can't get rid of these as they are needed for navigation in vaadin.com/docs. @pleku can verify. 

If those are needed for the book edition, maybe we can get rid of those using a script during build time or then use a different branch where all these are removed?

Reply:
I thought I had removed them all. I missed this one. Apologies